reana/reana_dev/helm.py

```diff
@@ -44,40 +45,42 @@ def helm_commands():
 @click.pass_context
 def helm_upgrade_components(ctx, user, push):  # noqa: D301
     """Upgrade REANA Helm dependencies."""
+    _check_if_missing_docker_releases()
+    new_docker_images = _get_docker_releases(user)
 
-    def _update_values_yaml(new_docker_images):
-        """Update all images in ``values.yaml``, skipping the ones up to date."""
-        values_yaml_relative_path = "helm/reana/values.yaml"
-        values_yaml_abs_path = os.path.join(
-            get_srcdir("reana"), values_yaml_relative_path
-        )
-        values_yaml = ""
-
-        with open(values_yaml_abs_path) as f:
-            values_yaml = f.read()
-            for docker_image in new_docker_images:
-                image_name, _ = docker_image.split(":")
-                if image_name in values_yaml:
-                    values_yaml = re.sub(
-                        f"{image_name}:.*", lambda _: docker_image, values_yaml, count=1
-                    )
-
-        with open(values_yaml_abs_path, "w") as f:
-            f.write(values_yaml)
-
-        display_message(
-            f"{values_yaml_relative_path} successfully updated.", component="reana"
-        )
+    values_yaml_abs_path = os.path.join(get_srcdir("reana"), "helm/reana/values.yaml")
+    _upgrade_docker_images(values_yaml_abs_path, new_docker_images)
+
+    prefetch_script_abs_path = os.path.join(
+        get_srcdir("reana"), "scripts/prefetch-images.sh"
+    )
+    _upgrade_docker_images(prefetch_script_abs_path, new_docker_images)
+
+    ctx.invoke(git_diff, component=["reana"])
+    if push:
+        git_push_to_origin(["reana"])
+
+
+def _get_docker_releases(dockerhub_user: str) -> List[str]:
```

Comment:
Can you please add a docstring?  The function is internal, so even a simple short one will do... It's always good to have some docstring e.g. for future refactor work.